javascript/presigned-url-example.ts

```diff
@@ -0,0 +1,140 @@
+import {MomentoSigner, CacheOperation} from '@gomomento/sdk';
+import fetch, {Response} from 'node-fetch';
+
+const signingKey = process.env.MOMENTO_SIGNING_KEY;
+if (!signingKey) {
+  throw new Error('Missing required environment variable MOMENTO_SIGNING_KEY');
+}
+const signingEndpoint = process.env.MOMENTO_ENDPOINT || '';
+if (signingEndpoint === '') {
+  throw new Error('Missing required environment variable MOMENTO_ENDPOINT');
+}
+
+let cacheName: string;
```

Comment:
style nit: can we make a helper function so that this can be a const instead of a let?

Reply:
also I apologize if the style stuff I'm commenting on is copypasta from the main example repo and not new things you've introduced.  Would be awesome to clean them both up if that's the case though, to prevent further copy/pasting.  I also think it might be worth having a `common` module that DRYs up the repetition between the two if there is a lot of it.  (I'm on the fence about that last part because obviously it's useful to be able to see all of the example code in one place as a user.)

Reply:
In this case it is different from the main example because in that example it actually creates the specified cache. I basically ported a lot of this from the [csharp example](https://github.com/momentohq/client-sdk-examples/blob/main/dotnet/MomentoExamples/MomentoApplicationPresignedUrl/Program.cs).

But yeah, I can move this into a helper function.